### DISCUSSION TITLE
Can I run RE on S5 Max that is currently running oroginal Valetudo?

### DISCUSSION THREAD

**Discussion by ando01** [rand256/valetudo#522] [General]
I have been trying to find this but cannot seem to find nay solid answers. I currently have an S5Max that I installed the original version of Valetudo on but want to run RE for some of the extra features. Has anyone successfully done this? If so, could you provide me with the steps to do it?

Thanks in advance,
Andy

**Comment by rand256** [comment]
Maybe you should check this link: https://github.com/rand256/valetudo/discussions/438
